### PR TITLE
circular-buffer: fix alignment of buffer representation

### PR DESCRIPTION
**Pull request by CePur**

### CODE REVIEW

**Comment by SleeplessByte** [comment]
Can you add the `prettier-ignore` range around these comments (See: https://prettier.io/docs/en/ignore.html#markdown). Otherwise, prettier (which will be added), will break this again in the future.

**Review by cmccandless — APPROVED** [review]

**Review by wolf99 — APPROVED** [review]

**Review by SleeplessByte — APPROVED** [review]

File: exercises/circular-buffer/description.md [hunk 1]
```diff
@@ -3,47 +3,47 @@ uses a single, fixed-size buffer as if it were connected end-to-end.
...
 are 1 & 2, leaving the buffer with just a 3:
-
+<!-- prettier-ignore -->
     [ ][ ][ ][ ][ ][3][ ]
 
 If the buffer has 7 elements then it is completely full:
-
-    [6][7][8][9][3][4][5]
+<!-- prettier-ignore -->
+    [5][6][7][8][9][3][4]
```
Comment (SleeplessByte) [inline exercises/circular-buffer/description.md:27]: Was 5 intended to move to the first position?

File: exercises/circular-buffer/description.md [hunk 2]
```diff
@@ -3,47 +3,47 @@ uses a single, fixed-size buffer as if it were connected end-to-end.
...
 When the buffer is full an error will be raised, alerting the client
 that further writes are blocked until a slot becomes free.
 
 When the buffer is full, the client can opt to overwrite the oldest
 data with a forced write. In this case, two more elements — A & B —
 are added and they overwrite the 3 & 4:
-
-    [6][7][8][9][A][B][5]
+<!-- prettier-ignore -->
+    [5][6][7][8][9][A][B]
```
Comment (SleeplessByte) [inline exercises/circular-buffer/description.md:36]: Was 5 intended to move to the first position?

File: exercises/circular-buffer/description.md [hunk 3]
```diff
@@ -3,47 +3,47 @@ uses a single, fixed-size buffer as if it were connected end-to-end.
...
 
 3 & 4 have been replaced by A & B making 5 now the oldest data in the
 buffer. Finally, if two elements are removed then what would be
 returned is 5 & 6 yielding the buffer:
-
-    [ ][7][8][9][A][B][ ]
+<!-- prettier-ignore -->
+    [ ][ ][7][8][9][A][B]
```
Comment (SleeplessByte) [inline exercises/circular-buffer/description.md:42]: Was "empty" intended to move to the first position?

File: exercises/circular-buffer/description.md [hunk 4]
```diff
@@ -3,47 +3,47 @@ uses a single, fixed-size buffer as if it were connected end-to-end.
...
 
 Because there is space available, if the client again uses overwrite
 to store C & D then the space where 5 & 6 were stored previously will
 be used not the location of 7 & 8. 7 is still the oldest element and
 the buffer is once again full.
-
-    [D][7][8][9][A][B][C]
+<!-- prettier-ignore -->
+    [C][D][7][8][9][A][B]
```
Comment (SleeplessByte) [inline exercises/circular-buffer/description.md:49]: Was C intended to move to the first position?

**Review by SleeplessByte — CHANGES_REQUESTED** [review]
Before this is merged, a few more questions.

File: exercises/circular-buffer/description.md (hunk 1 above)
Comment (CePur) [reply exercises/circular-buffer/description.md:27]: yes

File: exercises/circular-buffer/description.md (hunk 2 above)
Comment (CePur) [reply exercises/circular-buffer/description.md:36]: yes

File: exercises/circular-buffer/description.md (hunk 3 above)
Comment (CePur) [reply exercises/circular-buffer/description.md:42]: yes

File: exercises/circular-buffer/description.md (hunk 4 above)
Comment (CePur) [reply exercises/circular-buffer/description.md:49]: yes

**Review by SleeplessByte — APPROVED** [review]

**Comment by glennj** [comment]
I guess we're waiting on the v3 configlet improvements before we can conveniently regenerate the README for this practice exercise.

**Pull request merged**